toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -1117,8 +1124,9 @@ PreferencesPopup::PreferencesPopup()
 
   //--- Import/Export ------------------------------
   categoryList->addItem(tr("Import/Export"));
-  m_ffmpegPathFileFld = new DVGui::FileField(this, QString(""));
-  m_ffmpegTimeout     = new DVGui::IntLineEdit(this, 30, 1);
+  m_ffmpegPathFileFld       = new DVGui::FileField(this, QString(""));
+  m_fastRenderPathFileField = new DVGui::FileField(this, QString(""));
```

Comment:
Clicking "..." button (Open browser button) of this field takes extra time before opening the browser popup, since it tries to find non-existent path "desktop" as initial folder position.
In PR #853, I modified `DVGui::FileField`  constructor to be available to set `doNotBrowseInitialPath` flag for avoiding this behavior.

How about changing line 1128 like as follows?
`m_fastRenderPathFileField = new DVGui::FileField(this, QString("desktop"), false, true);`